tests/e2e/main.go

```diff
@@ -247,6 +282,22 @@ var rootCmd = &cobra.Command{
 			return err
 		}
 
+		decode := scheme.Codecs.UniversalDeserializer().Decode
+		secretData, err := ioutil.ReadFile("examples/clusterapi-apiserver-certs.yaml") // just pass the file name
```

Comment:
Since we have to create the secret with certificates, does it mean the secret was dropped from the mao template?

Reply:
yes, it's generated by the installer along with the others https://github.com/openshift/installer/blob/master/installer/pkg/config-generator/tls.go#L258